test/box/gh-5928-several-listen-sockets.lua

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env tarantool
+
+require('console').listen(os.getenv('ADMIN'))
+
+local function prepare_table_with_success_listen_uri(idx, default_server_addr)
+    assert(idx)
+    local table_uries = {
+        default_server_addr,
+        { string.format("%s", default_server_addr) },
+        { string.format("%sA", default_server_addr),
+          string.format("%sB", default_server_addr) },
+        { uri = string.format("%s", default_server_addr) },
+        { uri = string.format("%s", default_server_addr), transport = "plain" },
+        {
+          string.format("%sA", default_server_addr),
+          { uri = string.format("%s", default_server_addr), transport = "plain" }
+        },
+        {
+          { uri = string.format("%sA", default_server_addr), transport = "plain" },
+          { uri= string.format("%sB", default_server_addr), transport = "plain" }
+        },
+        { { uri = string.format("%s", default_server_addr), transport = {'plain', 'plain'} } },
+        { { uri = string.format("%s", default_server_addr), transport = 'plain, plain' } },
+        { { uri = string.format("%s", default_server_addr), transport = 'plain,,, plain' } },
+        default_server_addr .. "A, " .. default_server_addr .. "B",
+        default_server_addr .. "A?transport=plain, " .. default_server_addr .. "B?transport=plain",
+        default_server_addr .. "?transport=plain&transport=plain",
+        default_server_addr .. "A,,,,,, " .. default_server_addr .. "B", -- skip extra ','
+        -- skip extra '?', '&', '=' and ';'
```

Comment:
Should you? According to URI RFC, all question marks after the first one are considered regular symbols https://datatracker.ietf.org/doc/html/rfc3986#section-3.4. For example, if you have a query string `uri???key=?value?` then it means key = `??key` and value = `?value?`.

Reply:
Fixed, no extra delimiters is not ok